app/assets/stylesheets/hyrax/_forms.scss

```diff
@@ -72,6 +72,10 @@ form {
     }
   }
 
+  #collapseLease .form-group {
```

Comment:
This was necessary, and was the only way that we previously got the formatting correct for `#collapseEmbargo`, which is where this line was copied from.